src/components/data_comps/dlnd/dlnd_comp_mod.F90

```diff
@@ -385,10 +378,6 @@ subroutine dlnd_comp_run(EClock, x2l, l2x, &
     call t_startf('dlnd')
 
     call t_startf('dlnd_strdata_advance')
-    lsize = mct_avect_lsize(l2x)
-    do n = 1,lsize
-       l2x%rAttr(kf,n) = lfrac(n)
```

Comment:
This was a recent change that should not be removed.

Reply:
The value of kf will not be found in the mct version of seq_flds_mod. We are assuming that the base data model code (independent of the caps) should work for both mct and nuopc. So I will put this back in - but will do so in a way that queries whether this field is present in seq_flds_mod.